okhttp/src/main/resources/META-INF/proguard/okhttp3.pro

```diff
@@ -9,3 +9,4 @@
 
 # OkHttp platform used only on JVM and when Conscrypt dependency is available.
 -dontwarn okhttp3.internal.platform.ConscryptPlatform
+-dontwarn org.conscrypt.ConscryptHostnameVerifier
```

Comment:
strange; does dontwarn cover the files that shouldn’t warn, or the imports of those files? This file is only referenced from the file that precedes it

Reply:
I did an extra check. And I'm getting the following warning:
> Task :app:minify******ReleaseWithR8
AGPBI: {"kind":"warning","text":"Missing class: org.conscrypt.ConscryptHostnameVerifier","sources":[{}],"tool":"R8"}